### ISSUE TITLE
Server can't start without github package fully UP

### ISSUE THREAD

**Issue by Elikill58** [pterodactyl/panel#4061]
### Current Behavior

When github packages has problem, the server can't start because of infinite docker image loading.

### Expected Behavior

Ignoring it or timed out faster.
I know this is not useful for all time, but when there is a github issue, we are very happy to have something to prevent the server to be blocked before starting.

### Steps to Reproduce

1. Wait until github has problem such as today (https://www.githubstatus.com/):
![image](https://user-images.githubusercontent.com/17381066/165096807-13bd6b96-d298-4f23-a0b3-5f8d62b64550.png)
Or crate them yourself by cancelling all request to github's link
2. Try to start/restart server
3. See the server wait for long time that the github website timed out

### Panel Version

1.7.0

### Wings Version

-

### Games and/or Eggs Affected

Minecraft (Paper)

### Docker Image

ghcr.io/pterodactyl/yolks:java_8

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by DaneEveritt** [comment]
You're going to need to provide way more information, the only request to GitHub in the Paper egg is for a server properties file.

**Comment by parkervcp** [comment]
This looks like there is an issue when the git packages are down that it fails to pull images.

We check for updated images on start so it would hang while it waits.

**Comment by Boy132** [comment]
> You're going to need to provide way more information, the only request to GitHub in the Paper egg is for a server properties file.

Earlier today github packages were down and images hosted on ghcr.io couldn't be pulled. That prevented servers from starting.

**Comment by DaneEveritt** [comment]
So it failed to pull, but there are no error logs?

**Comment by Boy132** [comment]
There should be something in the wings logs. Here is one from a discord user that had this problem: https://ptero.co/ugybylyket
If the image doesn't exist locally and it can't be pulled, the container isn't created and thus the server doesn't start.

**Comment by Elikill58** [comment]
Sorry, personnally I didn't find special logs, maybe I was not searching in the good place. Also, sometimes it finally success to find the image, but after like 3 minutes. So no error, just lot of time. (without github issues it's like 1/2 seconds).

PS: Boy123 is totally right.

**Comment by matthewpi** [comment]
Gonna close this, if a container registry is down then of course we can't start servers.  If the image exists locally, it will use it until it can check the latest version.  If this is not how Wings actually behaves, then please open a new issue.
